levenshtein/zig/code.zig

```diff
@@ -16,6 +15,10 @@ fn levenshteinDistance(s1: []const u8, s2: []const u8, prev_row: []usize, curr_r
     const m = str1.len;
     const n = str2.len;
 
+    // Initialize arrays
+    var prev_row: [1024]usize = undefined;
+    var curr_row: [1024]usize = undefined;
```

Comment:
this is broken code and will panic with some inputs. C is doing VLAs which is also broken (e.g,. very large strings that overflow the stack).